region/maridia/inner-pink/West Cactus Alley.json

```diff
@@ -149,6 +152,7 @@
         ]},
         {"refill": ["Super", "Energy"]}
       ],
+      "flashSuitChecked": true,
```

Comment:
This looks like it should be deleted.  The strat above this too.
There are [1,1] and [2, 2] farmCycleDrops variations already.